papers/format.md

```diff
@@ -130,7 +130,7 @@ size | description
 size | description
 -----|------------------------------------
   4  | "INFO" block ID
-  4  | reserved
+  4  | size of the block excluding the ID and this field
```

Comment:
Will change to "size of this block" and insert the "excluding ID and this field" disclaimer in information to remove clutter.